azure-pipelines.yml

```diff
@@ -63,18 +69,21 @@ jobs:
       cd examples && python run_examples.py
     displayName: Examples
 
-- job: Windows
+- job: Windows_DPCPP
   pool:
-    vmImage: 'vs2017-win2016'
+    vmImage: 'windows-latest'
 
   steps:
+  - script: .ci/scripts/install_windows.bat $(WINDOWS_BASEKIT_URL) $(WINDOWS_DPCPP_COMPONENTS)
+    displayName: oneAPI environment
   - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
     displayName: Add conda to PATH
-
   - script: conda create -q -y -n CB python=3.7 conda-build conda-verify
     displayName: Create Anaconda environment
-
   - script: |
       call activate CB
+      set "DPCPPROOT=C:\Program Files (x86)\Intel\oneAPI\compiler\latest"
+      set "DAALROOT=C:\Program Files (x86)\Intel\oneAPI\daal\latest"
+      set "TBBROOT=C:\Program Files (x86)\Intel\oneAPI\tbb\latest"
```

Comment:
I would suggest separating build from testing in order to catch problems with the compiler RT. Because the user doesn't need runtime to run the cpu part.

Reply:
I think we should do both in ideal case. make build + testing with RT . and then do testing without RT.

Reply:
Why are you setting/assuming certain system paths here and not using the dal-devel and tbb-devel conda package paths in the python environment?

Reply:
Because there is no 2021 version of dal-devel or tbb in conda

- https://anaconda.org/intel/daal-devel/labels